Solutions/archTIS/Data Connectors/Logo/NCProtectLogo.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.0//EN" "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="body_1" width="200" height="19">
```

Comment:
Id should be guid.

Reply:
Hi @NikTripathi , Just wondering is data:image should be fine?

Reply:
According to parameters there should not be xlinks.

Reply:
Hi @NikTripathi , Thanks for your update. I am waiting for proper svg image without data:image as well as xlinks and it should be less than 5kb from business.